packages/mdc-switch/package.json

```diff
@@ -8,17 +8,22 @@
     "material design",
     "switch"
   ],
+  "main": "index.js",
```

Comment:
please point this to `dist/switch.js`

Reply:
Done

Reply:
I can't find a single other package we're doing this for... Can you explain this @moog16 ?

Reply:
Okay, I just found #3245, which explains this.

I don't think we should be rushing to do that just yet - please revert this here.

Reply:
sorry @rlfriedman! I jumped the gun :)